PyInstaller/hooks/hook-numpy.py

```diff
@@ -0,0 +1,15 @@
+import os
+import platform
+from PyInstaller.utils.hooks import collect_submodules, get_module_file_attribute
+
+# if we bundle the testing module, this will cause
```

Comment:
This "If wee bundle" sounds crooked. I suggest rewording into somethink like: "Including the testing module, this would …"